.pre-commit-config.yaml

```diff
@@ -1,10 +1,18 @@
 repos:
--   repo: https://github.com/ambv/black
-    rev: stable
-    hooks:
-    - id: black
-      language_version: python3.6
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v1.2.3
-    hooks:
-    - id: flake8
+- repo: local
+  hooks:
+  - id: isort
+    name: isort
+    entry: isort
+    language: python
+    types: [python]
```

Comment:
Not sure if `name` is required in these, actually, also `language` for a local repo. `entry` is the command to be run. `types` is the types of changed files it'll pass to the program as argument.